pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Monitor TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Monitor TCP sockets from Loopback interfaces:
+
+`nettop -m {{tcp}} -t {{loopback}}`
+
+- Monitor a specific process:
+
+`nettop -p {{process_id|process_name}}`
+
+- Display summary only for per-process:
+
+`nettop -P`
+
+- Monitor in logging mode and display 10 samples:
+
+`nettop -l {{10}}`
+
+- Monitor in delta mode, updating every 5 seconds:
```

Comment:
And instead of `delta mode` we could use:
```suggestion
- Monitor changes in the network, updating every 5 seconds:
```

Reply:
Sorry for leaving for a while because of other work...

How about `Monitor relative changes between every 5 seconds` ?
`delta` mode will show relative change, and default mode will show cumulative change.

I feel `delta mode` is a little bit hard to represent in a short words 😂

Reply:
The descriptions can be a little longer, as long as it has a purpose like displaying relevant information, keeping it short is nicer for the user but if it needs to be longer, it should be longer.

I wouldn't instantly correlate what delta mode is, but after thinking for around a minute, I remember it means change.

maybe `Monitor differences/changes between every 5 seconds`

Reply:
OK, so the description could be longer to make the presentation clearer. 

I think `Monitor changes between every 5 seconds` is good.

Reply:
I don't like `between`

Reply:
How about `during`?

Reply:
I think that's a case for @sbrl too.

Reply:
Just remove the between, it still makes complete sense

Reply:
Hello, @marchersimon, could you help out here, Do you think `Monitor changes every 5 seconds` can be applied?